Accessibility/Accessibility/ViewControllers/HomeShopVC.swift

```diff
@@ -7,24 +7,48 @@
 //
 
 import UIKit
+import URBNSwiftyConvenience
 
-class HomeShopVC: UIViewController {
-
+class HomeShopVC: UITableViewController {
+    private var categories: [String] = ["Mens", "Womens", "Home", "Sale"]
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        navigationItem.title = "Shop"
+        tableView.register(HomeShopTableViewCell.self, forCellReuseIdentifier: "categoryCell")
+        //whiteOut()
     }
     
+    func whiteOut() {
```

Comment:
I understand what you're trying to do here. But try to keep the PR's focused on the specific task. That way it makes it easier to review.